src/config/dynamic-theme-fixes.config

```diff
@@ -7638,8 +7638,11 @@ i.transfer-progress-icon
 span.top-icon.language
 
 CSS
-.files-grid-view.fm .column-settings {
-    background-color: ${#181a1b} !important;
+.grid-wrapper {
+    outline: 0px !important;
```

Comment:
Maybe changing the border color is bettter than removing it completely!
```suggestion
    outline-color: var(--darkreader-bg--surface-main);
```

Reply:
I agree with this one.

Reply:
Yup, even better!

Reply:
Great!, Thanks.